electrumsv/wallet.py

```diff
@@ -978,17 +993,31 @@ def get_address_history(self, address: Address):
                 if tx_fee is not None:
                     flags |= TxFlags.HasFee
                 updates.append((tx_hash, data, None, flags))
-            self._datastore.tx.update_or_add(updates)
+
+                self._datastore.tx.update_or_add(updates)
```

Comment:
Shouldn't this change be reverted? It looks (at least according to github) like it is accumulating updates and applying all the current ones repeatedly for each subsequent iteration, given indentation level.